cfgov/data_research/models.py

```diff
@@ -21,6 +20,158 @@ class ConferenceRegistration(models.Model):
     code = models.CharField(max_length=250)
 
 
+# mortgage metadata models
+
+class MortgageDataConstant(models.Model):
+    """Constant values that Research can change via the admin."""
+    name = models.CharField(max_length=255)
+    slug = models.CharField(max_length=255,
+                            blank=True,
+                            help_text="CAMELCASE VARIABLE NAME FOR JS")
+    value = models.IntegerField(null=True, blank=True)
+    string_value = models.TextField(blank=True)
+    note = models.TextField(blank=True)
+    updated = models.DateField(auto_now=True)
+
+    def __str__(self):
+        return "{} ({}), updated {}".format(self.name, self.slug, self.updated)
+
+    @classmethod
+    def get_thresholds(cls):
+        """Returns the constants we need to validate geo ares."""
+        threshold_count = cls.objects.get(name='threshold_count').value
+        threshold_year = cls.objects.get(name='threshold_year').value
+        return (threshold_count, threshold_year)
+
+    class Meta:
+        ordering = ['name']
+
+
+class MortgageMetaData(models.Model):
+    """
+    Metadata values, stored as json, and made available in the API.
+    """
+    name = models.CharField(max_length=255)
+    json_value = JSONField(blank=True)
+    note = models.TextField(blank=True)
+    updated = models.DateField(auto_now=True)
+
+    def __str__(self):
+        return "{}, updated {}".format(self.name, self.updated)
+
+    class Meta:
+        ordering = ['name']
+        verbose_name_plural = "Mortgage metadata"
+
+
+# mortgage geo models
+
+class State(models.Model):
+    fips = models.CharField(max_length=2, blank=True, db_index=True)
+    name = models.CharField(max_length=128, blank=True, db_index=True)
+    abbr = models.CharField(max_length=2)
+    ap_abbr = models.CharField(
+        max_length=20, help_text="The AP Stylebook's state abbreviation")
+    counties = JSONField(
+        blank=True, help_text='FIPS list of counties in the state')
+    non_msa_counties = JSONField(
+        blank=True,
+        help_text='FIPS list of counties in the state that are not in an MSA')
+    msas = JSONField(
+        blank=True, help_text='FIPS list of MSAs in the state')
+    non_msa_valid = models.BooleanField(default=False)
+
+    def __unicode__(self):
```

Comment:
In the metadata models above you're using `__str__`. That's more future-friendly. If you're running into unicode issues, you can use [Django's `python_2_unicode_compatible` decorator](https://docs.djangoproject.com/en/1.8/topics/python3/#str-and-unicode-methods). 

I think we should prefer `__str__` and use that decorator for all of the models using `__unicode__` to make our job of becoming Python3-compatible a little easier whenever that happens in the future.

Reply:
ack! `__str__` was my intention (I wrote model tests of `__str__`) but I slipped into old habits.